tests/parametric/test_dynamic_configuration.py

```diff
@@ -796,3 +808,62 @@ def test_trace_sampling_rules_with_tags(self, library_env, test_agent, test_libr
         span = trace[0]
         assert "_dd.p.dm" in span["meta"]
         assert span["meta"]["_dd.p.dm"] == "-12"
+
+    @parametrize("library_env", [{**DEFAULT_ENVVARS}])
+    def test_remote_sampling_rules_telemetry(self, library_env, test_agent, test_library):
+        """RC sampling rules should be reported by telemetry"""
+        event = set_and_wait_rc_telemetry(
+            test_agent,
+            config_overrides={
+                "tracing_sampling_rules": [
+                    {
+                        "service": "svc*",
+                        "resource": "*abc",
+                        "name": "op-??",
+                        "tags": [
+                            {"key": "tag-a", "value_glob": "ta-v*"},
+                            {"key": "tag-b", "value_glob": "tb-v?"},
+                            {"key": "tag-c", "value_glob": "tc-v"},
+                        ],
+                        "sample_rate": 0.5,
+                        "provenance": "dynamic",
+                    }
+                ],
+            },
+        )
+        configuration = event["payload"]["configuration"]
+        assert configuration == str(
+            [{"service": "svc*", "resource": "*abc", "name": "op-??", "sample_rate": 0.5, "provenance": "dynamic"}]
+        ) or str(
+            [
+                {
+                    "service": "svc*",
+                    "resource": "*abc",
+                    "name": "op-??",
+                    "tags": [{"tag-a": "ta-v*", "tag-b": "tb-v?", "tag-c": "tc-v"}],
+                    "sample_rate": 0.5,
+                    "provenance": "dynamic",
+                }
+            ]
+        )
+
+        event = set_and_wait_rc_telemetry(test_agent, config_overrides={"tracing_sampling_rules": []},)
+        configuration = event["payload"]["configuration"]
+        assert configuration == str([])
+
+    @parametrize("library_env", [{**DEFAULT_ENVVARS}])
+    def test_remote_sampling_rules_retention(self, library_env, test_agent, test_library):
+        """Only the last set of sampling rules should be applied"""
+        set_and_wait_rc(
+            test_agent, config_overrides={"tracing_sampling_rules": [{"service": "svc*", "sample_rate": 0.5}],}
```

Comment:
The RFC says that remote sampling rules will always have a `provenance`, so we should add one to each of these test rules to be consistent